autogen/agentchat/groupchat.py

```diff
@@ -42,12 +42,22 @@ class GroupChat:
         - "random": the next speaker is selected randomly.
         - "round_robin": the next speaker is selected in a round robin fashion, i.e., iterating in the same order as provided in `agents`.
 
-    - allow_repeat_speaker: whether to allow the same speaker to speak consecutively. Default is True, in which case all speakers are allowed to speak consecutively. If allow_repeat_speaker is a list of Agents, then only those listed agents are allowed to repeat. If set to False, then no speakers are allowed to repeat. allow_repeat_speaker and allowed_or_disallowed_speaker_transitions are mutually exclusive.
-    - allowed_or_disallowed_speaker_transitions: a dictionary of keys and list as values. The keys are the source agents, and the values are the agents that the key agent can transition to. Default is None, in which case a fully connected allowed_speaker_transitions_dict is assumed. allow_repeat_speaker and allowed_or_disallowed_speaker_transitions are mutually exclusive.
-    - speaker_transitions_type: whether the speaker_transitions_type is a dictionary containing lists of allowed agents or disallowed agents. allowed means the allowed_or_disallowed_speaker_transitions is a dictionary containing lists of allowed agents. If set to disallowed, then the allowed_or_disallowed_speaker_transitions is a dictionary containing lists of disallowed agents. Must be supplied if allowed_or_disallowed_speaker_transitions is not None.
+    - allow_repeat_speaker: whether to allow the same speaker to speak consecutively.
+        Default is True, in which case all speakers are allowed to speak consecutively.
+        If `allow_repeat_speaker` is a list of Agents, then only those listed agents are allowed to repeat.
+        If set to False, then no speakers are allowed to repeat.
+        `allow_repeat_speaker` and `allowed_or_disallowed_speaker_transitions` are mutually exclusive.
+    - allowed_or_disallowed_speaker_transitions: a dictionary of keys and list as values.
+        The keys are the source agents, and the values are the agents that the key agent can transition to.
```

Comment:
The keys are source agents, and the values are agents that the key agent can transit to.

Reply:
revised.